### PR TITLE
Translation PR

### PR DESCRIPTION
**Pull request by guacbot**
# Translations
This is an automated PR created by the scheduled translation task pulling the latest translated files
## Preview Site
[/guacbot/translation-20190127042713/](/guacbot/translation-20190127042713/)

### CODE REVIEW

File: content/integrations/amazon_web_services.fr.md [hunk 1]
```diff
@@ -2,7 +2,8 @@
 aliases:
   - /fr/integrations/aws/
   - /fr/logs/aws
-  - /fr/integrations/faq/revoking-aws-keys-and-enabling-role-delegation-for-the-datadog-aws-integration/
+  - >-
+    /fr/integrations/faq/revoking-aws-keys-and-enabling-role-delegation-for-the-datadog-aws-integration/
```
Comment (davidejones) [inline content/integrations/amazon_web_services.fr.md]: @l0k0ms This appears to be valid yaml and is something called `block chomping indicator` 

https://stackoverflow.com/a/3790497/1113788
https://yaml.org/spec/1.2/spec.html#id2794534

As long as it didn't cause hugo to fail it should be fine for now?

Comment (l0k0ms) [reply content/integrations/amazon_web_services.fr.md]: yup, didn't know that thanks for the info 🙇

Comment (davidejones) [reply content/integrations/amazon_web_services.fr.md]: I found a setting that should reduce the chances of this happening by increasing line length. I'm releasing it now

Comment (l0k0ms) [reply content/integrations/amazon_web_services.fr.md]: 🙇

**Pull request closed without merging**